bears/vcs/git/GitCommitBear.py

```diff
@@ -134,6 +145,34 @@ def run(self, allow_empty_commit_message: bool = False, **kwargs):
             body,
             **self.get_issue_checks_metadata().filter_parameters(kwargs))
 
+    def check_auth_email(self, auth_email,
+                         auth_email_check: bool=True,
+                         email_dns_check: bool=True):
+        """
+        Checks validity of author's email
+
+        :param auth_email_check:      If True, it will check validity of
+                                      author's email.
+        :param email_dns_check:       If True, it will confirm the domain
+                                      in author's email is valid.
+        """
+        if not auth_email_check:
+            return
+
+        # Used regex provided in http://emailregex.com/
+        email_check = re.match(r'(^[a-zA-Z0-9_.+-]'
+                               + r'+@[a-zA-Z0-9-]+\.'
+                               + r'[a-zA-Z0-9-.]+$)', auth_email)
+
+        # if auth_email do not match regex there is no need to
+        # look up dns
+        if email_dns_check and email_check:
+            email_check = not is_email(auth_email, diagnose=True,
+                                       check_dns=True).ERROR_CODES
+        if not email_check:
+            yield Result(self,
+                        'Commit\'s author has invalid email.')
```

Comment:
The code does not comply to PEP8.

*PEP8Bear, severity NORMAL, section `autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/bears/vcs/git/GitCommitBear.py
+++ b/bears/vcs/git/GitCommitBear.py
@@ -171,7 +171,7 @@
                                        check_dns=True).ERROR_CODES
         if not email_check:
             yield Result(self,
-                        'Commit\'s author has invalid email.')
+                         'Commit\'s author has invalid email.')
 
     def check_shortlog(self, shortlog,
                        shortlog_length: int=50,
```